docs/BLTouch.md

```diff
@@ -1,50 +1,129 @@
+Now, it's time to test homing with a twist. Instead of letting the probe pin touch the print bed,
+let it touch the nail on your finger. So issue a `G28`, wait until it starts to move down, and stop
+the movement by very gently touching the pin with your nail. You probably have to do it twice, since
+the default configuration makes it probe twice. But be prepared to turn off the printer, to avoid
+damage, if it doesn't stop when you touch the pin.
 
-    GET_POSITION
+If that was successful, do another `G28` but this time let it touch the bed as it should.
 
-In the output lines the `toolhead` line with the `Z:` contains the required value.
+Calibrating the BL-Touch
+========================
 
 ### X/Y Offset
-In order to find the propper vertical offset of the probe you need to know a certain point of
+
+In order to make klipper work properly you need to tell it in which relation to the nozzle
+the probe is exactly located. Lets start with the `x_offset` and the `y_offset`
+
+In order to find the proper vertical offset of the probe you need to know a certain point of
 your bed. X/Y = 0 may be a good point for this, any other will do as long as you know it.
-Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removeable mark on your
+Find it by lowering the nozzle next to it using `g0 z0.5`. Create a removable mark on your
 bed by i.e. using a non permanent marker.
 
-Now move the tip of the BL-Touch over that point by jogging there with the conrols in the `control` tab of
+Now move the tip of the BL-Touch over that point by jogging there with the controls in the `control` tab of
 OctoPrint. Once the BL-Touch is roughly over the point acquire it using the `GET_POSITION` command.
 The difference to your marked point is your `x_offset` and `y_offset` to configure in the `printer.cfg`.
 
 
+### Z Offset
+
+1. We start by changing the `z_offset` in the `bltouch` section of the configuration to 10. This
+setting is very wrong, but it will make it possible to move down to the right height using the
+menus. Run `RESTART` after changing it to reload the configuration.
+
+2. Now run `G28`, to home the printer. Take a note of the point where the pin hits the bed and use
+the menu to move the nozzle to that point. Now take a *folded* paper and put it under the nozzle.
+Move the nozzle down using the menu, in 0.1mm steps, until it grabs the paper. It doesn't matter how
+much it grabs, you are searching for the higest position that grabs the paper. The nozzle should now
+be between 0.1mm and 0.2mm above the printbed. If you wonder why we used a folded paper, the answer
+is that the granularity of the movement is only 0.1mm, or the same as a normal paper thickness, that
+means that you could easily move too far and hit the bed.
+
+3. Now update the `z_offset` to reflect this, by substracting the current offset (10) by the reading
+on the printer display. So if it says 8.2, then `10-8.2` = 1.8
```

Comment:
the printer doesn't have to have a display. the GET_POSITION will allways work.

Reply:
There are other statements too, that assume that you have a display and physical controls. I will change everything to not make those assumptions.